check-sorting.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
nicely done :)

Would you mind adding an explanation of this to the README?

Reply:
Sure :)